### PR TITLE
Added front-end boilerplate to the base project

### PR DESCRIPTION
**Pull request by jamonholmgren**
Explanation in README.md

@yulianglukhenko @ryanlntn @ajrob27

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,10 +45,34 @@ Since it is likely you'll want to change the name of this project after you've c
...
+
+PhoenixBase comes with a basic Bitters setup for front end HTML/CSS and grid. You can learn more about it
+at the links below, but here is an intro.
+
+### Basic setup
+
+1. Set up your grid in web/static/css/base/_grid-settings.sass (uses [Bourbon Neat](http://neat.bourbon.io))
+2. In the same folder, set up global variables (colors, etc) in _variables.scss
+3. Put custom fonts in assets/fonts and import in web/static/css/base/_fonts.sass
+4. Edit your app, header, and footer templates in web/templates/layout
```
Comment (ajrobertsonio) [inline README.md:58]: No comma after "header"

**Comment by ajrobertsonio** [comment]
Looks awesome! Nice work @jamonholmgren 👍

**Comment by ryanlntn** [comment]
@jamonholmgren Looks like `has_flash?` wasn't a good name after all:

> 10: PredicateName: Favour `flash?` over `has_flash?`

**Comment by ryanlntn** [comment]
Someone should respond with favor "favor" over "favour".

**Comment by jamonholmgren** [comment]
@ryanlntn Where did you see that? Funny that we stumbled into that particular issue.

**Comment by jamonholmgren** [comment]
🚢 ed

**Pull request merged**